backend/main/server/helpers.go

```diff
@@ -476,7 +476,19 @@ func (h *Helpers) enforceCommunityResitrictions(
 			return errors.New(errMsg)
 		}
 	} else {
-		hasBalance, err := h.processTokenThreshold(p.Creator_addr, s)
+		threshold, err := strconv.ParseFloat(*c.Proposal_threshold, 32)
+		if err != nil {
```

Comment:
shouldn't this be 64 for `bitSize` param? `shared.Contract.Threshold` is of type `float64` ? Or some specific reason here why it's set to 32?

Reply:
Good catch - it is probably a copy/paste error when I was looking up parsing floats from strings.

Reply:
another PR fixes this